source/installation_guide/upgrading/from-2.x-to-3.0.rst

```diff
@@ -0,0 +1,37 @@
+=====================================
+Upgrading Dovecot from 2.x to 3.0/2.4
+=====================================
+
+.. contents::
+   :depth: 1
+   :local:
+
+
+About version numbers
+=====================
+
+With 3.0 / 2.4 release Pro and CE release have different major version numbers.
+In documentation any reference to 3.0 applies to 2.4 as well.
+
+Recommended upgrade path
+========================
+
+The recommended upgrade path for 3.0/2.4 is to perform a migration to the version.
+We do not recommend in-place upgrade due to significant number of removed and changed
+features. Performing migration is the safest way to start using 3.0/2.4 version.
+
+.. seealso::
+  :ref:`migrating_mailboxes`.
+
+Removed features and their replacements
+=======================================
+
+Quota
+-----
+
+ - The dict and dirsize backends are removed.
+   You should use :ref:`count <quota_backend_count>` instead along with
+   :ref:`quota_clone plugin <quota_clone_plugin>`.
+
+   Note that using quota clone plugin can cause all user's indexes to update,
+   so reserve time for this.
```

Comment:
I don't think quota_clone plugin causes any extra index updates. It's the changing to count backend that can cause it.

Reply:
Duh, was supposed to write count...